libvirtd-desktop/README.md

```diff
@@ -7,13 +7,43 @@
   ```
   $ sudo systemd-sysusers /usr/lib/sysusers.d/libvirt-qemu.conf
   ```
+- Create the `libvirtdbus` user and group:
```

Comment:
Let's use systemd-sysusers config instead

Reply:
There's not an existing config to create the libvirt group & libvirtd user/group via `systemd-sysusers` in `fedora-sysexts/libvirtd-desktop/rootfs/usr/lib/sysusers.d/` dir, (only clevis.conf & libvirt-qemu.conf) but I'd be glad to make one.

Here's what I came up with:
```text
$ cat ./libvirt-daemon-sysusers.conf
g libvirt -

$ sudo systemd-sysusers ./libvirt-daemon-sysusers.conf
Creating group 'libvirt' with GID 963.

$ cat ./libvirt-dbus-sysusers.conf
u libvirtdbus - "Libvirt D-Bus bridge" / /sbin/nologin

$ sudo systemd-sysusers ./libvirt-dbus-sysusers.conf
Creating group 'libvirtdbus' with GID 964.
Creating user 'libvirtdbus' (Libvirt D-Bus bridge) with UID 964 and GID 964.
```

If we go this route, would the plan be to add the actual file creation step into the Justfile (like the manual work for netcat), with a destination of `rootfs/usr/lib/sysusers.d/`, so the files are included in the sysext .raw package?

Reply:
Ideally those should be provided upstream so we should report that there somewhere. Not sure why they are not already. We can include them here in the meantime (you can use the "files" option in the justfile to add them and the COPY one in the containerfile).

Reply:
See the iwd sysext for an example: https://github.com/travier/fedora-sysexts/tree/main/iwd